keras_cv/models/backbones/resnet_v1/resnet_v1_backbone_presets_test.py

```diff
@@ -63,10 +59,9 @@ def test_backbone_output_with_weights(self):
         # We should only update these numbers if we are updating a weights
         # file, or have found a discrepancy with the upstream source.
 
-        outputs = feature_extractor(tf.ones(1, 512, 512, 3))
```

Comment:
This fix seems correct to me -- as is, this test should fail because it's an invalid call to `tf.ones`